tailwind.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       default: {
         checkbox: {
           icon:
-            '<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="12" height="12" rx="2" fill="#8ACC9E"/></svg>'
+            '<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="12" height="12" rx="2" fill="#ffffff"/></svg>'
```

Comment:
That's how checked checkbox represented in the [Figma mock-up](https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=590%3A216):
![image](https://user-images.githubusercontent.com/140053/87783531-51d9a600-c83d-11ea-97bd-4108c809318f.png)
That's how it looks in the browser:
![image](https://user-images.githubusercontent.com/140053/87783557-60c05880-c83d-11ea-8727-69385c7907ca.png)
There should be a green square in the middle of the grey square, not the opposite. BTW the same is for the radio button. It looks like we just messed the styles a bit...

To achieve this behaviour you can try something like this in `tailwind.config.js` around line 11:

```js
checkbox: {
  width: theme('spacing.6'),
  height: theme('spacing.6'),
  icon: '<svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">' +
    '<rect x="0.5" y="0.5" width="21" height="21" rx="2.5" fill="white" stroke="#E7E7E7"/>' +
    '<rect x="5" y="5" width="12" height="12" rx="2" fill="#8ACC9E"/>' +
    '</svg>'
},
radio: {
  width: theme('spacing.6'),
  height: theme('spacing.6'),
  icon: '<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">' +
    '<circle cx="12" cy="12" r="11.5" fill="white" stroke="#E7E7E7"/>' +
    '<path d="M18 12C18 15.3137 15.3137 18 12 18C8.68629 18 6 15.3137 6 12C6 8.68629 8.68629 6 12 6C15.3137 6 18 8.68629 18 12Z" fill="#8ACC9E"/>' +
    '</svg>'
}
```

I exported the SVGs from Figma. And used them in the Tailwind Custom Forms config, [here](https://tailwindcss-custom-forms.netlify.app/)'s the documentation.

Reply:
Hi @amiskov, I spoke with Helton about the style of the button because the figma style (green fill and "white" border) causes an effect of being a smaller square due to the page's background color. He told me to use the other way around.

Reply:
Hmmm, this should be updated on Figma then, or at least be spoken of in #ux-ui so everyone knows what's going on. We should expect communication problems to happen as we grow larger. 

@heltonlr can you please update the Figma designs?

And for everyone else, please lets try to discuss design stuff that don't make sense in shared channels so we have a way to avoid situations like this

Reply:
Sorry, I'll write in Portuguese so I don't have any doubts:

@victorgdev , acho que eu te expliquei errado. O checkbox continua como está, com o quadrado verde indicando que o mesmo está selecionado. Já a borda em cinza é que podemos escurecer para aparecer melhor no background daquela página. Se quiser, me liga para esclarecer melhor essa questão. Desculpa qualquer confusão.

Reply:
@heltonlr sorry for the miscommunication and I will revert the configurations of the checkbox! Thanks!